app/Console/Commands/ClearLogs.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class ClearLogs extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'logs:clear';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Clear log files';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        unlink('storage/logs/laravel.log');
```

Comment:
In looking at this, `unlink()` will delete the file, which normally would be fine if the permissions on the `storage/logs` directory are right, but filesystems and groups/users permissions can be setup wrong sometimes, so we might want to use something more like `ftruncate()` with `rewind()`.  Otherwise we're deleting the file, and if the web server doesn't have permission to *create* files within that directory, it could cause other weird errors.

https://www.php.net/manual/en/function.ftruncate.php